manifests/secondarydns.yaml

```diff
@@ -178,3 +178,60 @@ spec:
           name: secondary-dns
       - name: secdns-zones
         emptyDir: {}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ksd-allow-egress-to-dns
+  namespace: secondary
+spec:
+  podSelector:
+    matchLabels:
+      k8s-app: secondary-dns
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: kube-system
+      podSelector:
+        matchLabels:
+          k8s-app: kube-dns
+    ports:
+    - protocol: TCP
+      port: dns-tcp
+    - protocol: UDP
+      port: dns
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ksd-allow-egress-to-api-server
+  namespace: secondary
+spec:
+  podSelector:
+    matchLabels:
+      k8s-app: secondary-dns
+  policyTypes:
+  - Egress
+  egress:
+  - ports:
+    - protocol: TCP
+      port: 6443
```

Comment:
maybe worth comments that this k8s API port ?

same on https://github.com/kubevirt/kubesecondarydns/pull/99/files?diff=unified&w=0#r2097292314 once answered please

Reply:
> maybe worth comments that this k8s API port ?

Please note that the policy name already indicate its about the affecting connectivity to kube-api server
Other policies follow the same naming.